Sources/xcresultparser/JunitXML.swift

```diff
@@ -100,16 +102,11 @@ public struct JunitXML {
         
         if testReportFormat != .sonar {
             if let date = testAction?.startedTime {
-                let dateFormatter = ISO8601DateFormatter()
-                testsuites.addAttribute(name: "timestamp", stringValue: dateFormatter.string(from: date))
```

Comment:
We do not want to add the startetTime anymore? Only the duration? If so, we could combine the if statements.
```
if let date = testAction?.startedTime,
           let ended = testAction?.endedTime {
    let duration = ended.timeIntervalSince(date)
    testsuites.addAttribute(name: "time", stringValue: numFormatter.unwrappedString(for: duration))
}

```

Reply:
Disregard the question. I suppose the answer here is also: "Fix junit output format to conform to the XSD".

Reply:
yep correct as per https://github.com/junit-team/junit5/blob/main/platform-tests/src/test/resources/jenkins-junit.xsd#L109 only time is an allowed property.
> time is the time that was required to process the test rules